src/AppEntry.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from './utilities/store';
+import Router from './router';
+import { IntlProvider } from 'react-intl';
+
+const AppEntry = () => (
+  <Provider store={store(true)}>
+    <IntlProvider locale="en">
+      <Router />
+    </IntlProvider>
+  </Provider>
+);
+
+/**
+ * Register simple service worker JS assets pre chache
+ * If registration fails, the app will continue to work withouth any issues.
+ * Service worker is generated by webpack and hould be ever avaiable only in production version of the app.
+ * You don't yant your JS bundles to be cached in dev env.
+ * You must test it locally in non HTTPS environment due to the insecure origin and self signed certificate issues cause by the PROXY.
+ * These cannot be overcome due to xss security policy inside service workers browser implementaion to avoid potential securitu issues.
+ */
+if (navigator.serviceWorker) {
```

Comment:
You no longer need the service worker. It's not generated anymore and the chrome 2 performance improvement makes it not really worth it.